### PR TITLE
Add selector for theme manager preview

### PR DESCRIPTION
**Pull request by timja**
See https://github.com/jenkinsci/theme-manager-plugin/pull/75

Integrates with up-coming theme manager feature so that we can show the colours in the preview

image

### Proposed changelog entries

* Entry 1: (skip, not relevant till theme manager ships)
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- ~~New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate.~~
- ~~For dependency updates: links to external changelogs and, if possible, full diffs~~

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- ~~If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))~~
- ~~If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).~~

### CODE REVIEW

**Review by janfaracik — APPROVED** [review]

File: war/src/main/less/abstracts/theme.less
Comment (daniel-beck) [inline war/src/main/less/abstracts/theme.less:1]: Probably should have a comment indicating why this exists? This looks like one of those random undocumented things that'll be gone the next time someone restructures something.

**Review by uhafner — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by MarkEWaite — APPROVED** [review]

**Pull request merged**
